rmgpy/cantherm/input.py

```diff
@@ -75,7 +79,72 @@
 jobList = []
 
 ################################################################################
+def database(
+             thermoLibraries = None,
+             transportLibraries = None,
+             reactionLibraries = None,
+             frequenciesLibraries = None,
+             kineticsFamilies = 'default',
+             kineticsDepositories = 'default',
+             kineticsEstimator = 'rate rules',
+             ):
+    # This function just stores the information about the database to be loaded
+    # We don't actually load the database until after we're finished reading
+    # the input file
+    if isinstance(thermoLibraries, str): thermoLibraries = [thermoLibraries]
+    if isinstance(transportLibraries, str): transportLibraries = [transportLibraries]
+    if isinstance(reactionLibraries, str): reactionLibraries = [reactionLibraries]
+    if isinstance(frequenciesLibraries, str): frequenciesLibraries = [frequenciesLibraries]
+    databaseDirectory = settings['database.directory']
+    thermoLibraries = thermoLibraries or []
+    transportLibraries = transportLibraries
+    # Modify reactionLibraries if the user didn't specify tuple input
+    if reactionLibraries:
+        index = 0
+        while index < len(reactionLibraries):
```

Comment:
Why not `for index in xrange(len(reactionLibraries))`?
(I understand that's the input in `rmgpy/input.py`)